routes/forum.js

```diff
@@ -83,6 +105,10 @@ module.exports = function(app) {
   };
 
   var handleThreadCreate = function(req, res) {
+
+    if(req.session.user)
```

Comment:
I recommend using curly brackets all the time, even if your if-statement only has 1 line of code in it.